client/state/jetpack-connect/actions.js

```diff
@@ -348,23 +351,24 @@ export default {
 				} );
 				// Update the user now that we are fully connected.
 				userFactory().fetch();
-				return wpcom.me().sites( { site_visibility: 'all', include_domain_only: true } );
+				dispatch( {
+					type: SITE_REQUEST,
+					siteId
+				} );
+				return wpcom.site( siteId ).get();
 			} )
 			.then( ( data ) => {
 				tracksEvent( dispatch, 'calypso_jpc_auth_sitesrefresh', {
```

Comment:
I'm going to leave it as is, just in case we are using the event in any funnels.